documentation/assemblies/deploying/assembly-deploy-create-cluster.adoc

```diff
@@ -17,8 +17,7 @@ But you can deploy and use the Topic Operator and User Operator as standalone co
 NOTE: The Cluster Operator can watch one, multiple, or all namespaces in a Kubernetes cluster.
 The Topic Operator and User Operator watch for `KafkaTopics` and `KafkaUsers` in the single namespace of the Kafka cluster deployment.
 
-[discrete]
-== Deploying a Kafka cluster with the Topic Operator and User Operator
+.Deploying a Kafka cluster with the Topic Operator and User Operator
```

Comment:
Will this keep these titles linkable? I think we really want to be able to deep link to them.

Reply:
They weren't linkable with the [discrete]. If you think that's important, I've set as level three headings without discrete.